docs/user-guide/local.md

```diff
@@ -0,0 +1,207 @@
+---
+layout: main
+title: Local Build
+category: User Guide
+menu: menu
+toc:
+- title: What is sd-local ?
+  url: "#what-is-sd-local-?"
+- title: Installing sd-local
+  url: "#installing-sd-local"
+- title: Simple usage
+  url: "#simple-usage"
+- title: config command
+  url: "#config-command"
+- title: build command
+  url: "#build-command"
+---
+
+# What is sd-local ?
+sd-local enables running a more feature-complete local Screwdriver without needing uploading source code to SCM. With sd-local, you can use features such as Command, Template, Metadata, and more.
+
+
+# Installing sd-local
+## Requirements
+- [docker](https://www.docker.com/)
+  - Use Docker container as build environment.
+- [git](https://git-scm.com/)
+  - Required when using [--src-url] option.
```

Comment:
Is this `[src-url]` intended to a link?  Or should we using backquotes?

Reply:
I fixed.